developers/weaviate/starter-guides/managing-collections/collections-scaling-limits.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Do you have too many collections?
+sidebar_label: Scaling limits with collections
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+# tags: ['performance']
+---
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+:::caution You hit the collections count limit?
```

Comment:
Curious - what was the rationale about starting with this admonition?

Given this is a starter guide, I am not super sure about starting with this admonition, as it reads like a troubleshooting guide.

Reply:
As the page is linked in the Weaviate error message, it's important for the user to be aware of the context instantly. After checking out the rest of the comments, I will think about possible solutions.